census/estimator/requirements.txt

```diff
@@ -0,0 +1,4 @@
+google-cloud-dataflow
+apache-beam[gcp]
+tensorflow==1.10.0
```

Comment:
CMLE supports TF 1.12 does this be a better version than 1.10?

Reply:
Done